Model/Category.php

```diff
@@ -62,6 +97,8 @@ public function getName()
     public function setEnabled($enabled)
     {
         $this->enabled = $enabled;
+
+        return $this;
```

Comment:
AFAIK, nothing says that self should be returned on the interface.

BTW, this is not related to the PR (fix php doc). If you want to improve it, submit it on another one.